### PR TITLE
v8.1 pre-alpha

### PR DESCRIPTION
**Pull request by nullsystem**

### CODE REVIEW

File: src/blog/2024-09-19/index.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+Enabled by default using the server-side convar `neo_sv_suicide_prevent_cap_punish 1`.
+It will attempt to deal with multiple of scenarios:
+
+* Last-person suicide (`kill` command, grenade, or by environment)
+* Last-person disconnect/timeout then reconnect
+* Posthumous teamkill: Throw a grenade at your teammate then suicide, final-kill attribute to teammate
+
+## Bug Fixes
+
+* Fix issue where spread would be halved
```
Comment (brysondev) [inline src/blog/2024-09-19/index.md]:
```suggestion
* Fixed issue where spread would be halved
```

File: src/blog/2024-09-19/index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+* Main menu changes
+    * Change normal font from Neuropol2 to Green Mountain for readability
```
Comment (brysondev) [inline src/blog/2024-09-19/index.md]:
```suggestion
    * Changed normal font from Neuropol2 to Green Mountain for readability
```

File: src/blog/2024-09-19/index.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    * Settings confirmation popup add "Cancel" button
```
Comment (brysondev) [inline src/blog/2024-09-19/index.md]:
```suggestion
    * Settings confirmation popup now has a "Cancel" button
```

File: src/blog/2024-09-19/index.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+### First person muzzle flashes
+
+Muzzle flashes now shown in first-persons and tracers are disabled.
+
+### Ghost cap prevention
+
+Ghost cap prevention awards XPs to oppositing team if a player tries to prevent a ghost cap.
+Enabled by default using the server-side convar `neo_sv_suicide_prevent_cap_punish 1`.
+It will attempt to deal with multiple of scenarios:
```
Comment (AdamTadeusz) [inline src/blog/2024-09-19/index.md:24]: don't belittle yourself, "It will deal with multiple scenarios:"

**Pull request merged**